modelservice/pom.xml

```diff
@@ -83,6 +83,16 @@
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
 
+        <!-- Swagger -->
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger-ui</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+        </dependency>
+
```

Comment:
Please use the commit/runtime/test scheme for grouping the dependencies. (We were originally using the library names but decided to go with the type.)